src/helics/MessageFederates.hpp

```diff
@@ -0,0 +1 @@
+

```

Comment:
This file appears to be empty. What is the purpose of this file?

Reply:
I am not quite done yet, but this will be a convenience header to include all the files related to MessageFederates in the application API